ui/build/build.api.js

```diff
@@ -117,6 +118,11 @@ const objectTypes = {
     required: [ 'desc' ]
   },
 
+  meta: {
+    props: [ 'docsRoute', 'docsPage', 'docsApiAnchor' ],
```

Comment:
Let's use one prop only "docsUrl" with full URL. For now, point to `https://v1.quasar.dev`. Example:

```js
meta: {
  docsUrl: 'https://v1.quasar.dev'
}
```